### PR TITLE
modify build/configuration guidance for docker

### PR DESCRIPTION
**Pull request by sunghan-chang**
1. README: add dot at numbering
2. os/tools/configure: guide dbuild.sh to execute distclean

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d25059606e465e02f9f85d351620d88f4ad985ba] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d25059606e465e02f9f85d351620d88f4ad985ba] - Code Rule Check OK.

File: os/tools/configure.sh [hunk 1]
```diff
@@ -153,7 +153,7 @@ fi
 if [ -r ${dest_config} ]; then
   if [ -r "${TOPDIR}/.version" ]; then
     echo "Already configured and compiled!"
-    echo "Do 'make distclean' and try again."
+    echo "Do './dbuild.sh distclean' and try again."
```
Comment (lokeshbv) [inline os/tools/configure.sh]: what if one doesn't want to use docker build env

Comment (sunghan-chang) [reply os/tools/configure.sh]: Yeah, so we added "otherwise ...".

**Comment by seinfra** [comment]
Target : [4d5bc93a0b4cb0552e9e328762a4af2900bc157d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4d5bc93a0b4cb0552e9e328762a4af2900bc157d] - Code Rule Check (C++) OK.

**Pull request merged**